scripts/linters/html_linter.py

```diff
@@ -345,24 +345,22 @@ def lint_html_files(self):
         error_messages = []
         full_error_messages = []
         htmllint_cmd_args = [node_path, htmllint_path, '--rc=.htmllintrc']
-        html_files_to_lint = self.html_filepaths
-        for filepath in html_files_to_lint:
-            proc_args = htmllint_cmd_args + [filepath]
-            proc = subprocess.Popen(
-                proc_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
-            encoded_linter_stdout, _ = proc.communicate()
-            linter_stdout = encoded_linter_stdout.decode(encoding='utf-8')
-            # This line splits the output of the linter and extracts digits
-            # from it. The digits are stored in a list. The second last
-            # digit in the list represents the number of errors in the file.
-            error_count = (
-                [int(s) for s in linter_stdout.split() if s.isdigit()][-2])
-            if error_count:
-                failed = True
-                full_error_messages.append(linter_stdout)
-                error_messages.append(
-                    self._get_trimmed_error_output(linter_stdout))
+        proc_args = htmllint_cmd_args + self.all_filepaths
```

Comment:
Going by the older implementation, shouldn't this line be the following:
```suggestion
        proc_args = htmllint_cmd_args + self.html_filepaths
```
i.e `self.html_filepaths` instead of `self.all_filepaths`, or is there a reason for doing this change?

Reply:
Thanks, @sajalasati Both works the same but as you have asked, I have replaced all_filepath with html_filepaths.

Reply:
My bad I realised that later only, both are the same, so even the older code was good enough. (Though in future, if you feel your implementation is correct, you should first argue about it and check if that doesn't meet the requirement or the reviewer is missing something. Don't jump to directly making the change since reviewer asked you.)

Reply:
okay.